.ci_support/migrations/azure_storage_202409.yaml

```diff
@@ -0,0 +1,19 @@
+__migrator:
+  build_number: 1
+  commit_message: Rebuild for Azure Storage September release
+  kind: version
+  migration_number: 1
+  exclude_pinned_pkgs: false
+azure_storage_common_cpp:
+- 12.8.0
+azure_storage_blobs:
```

Comment:
I found the problem. I omitted the `_cpp` part of the conda package name

Reply:
Fixed by applying https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6480 and rerendering